content/2025/china/_index.html

```diff
@@ -6,19 +6,32 @@
 hide_sidebar: true
 hide_breadcrumb: true
 container: "container-fluid"
-tagline: "25 July 2025"
+tagline: "25 July 2025, Bvstin Hotel, ChengDu"
 layout: "single"
+tagline_suffix: |
+  <a class="btn btn-bordered padded fw-700 margin-top-40 margin-bottom-50" href="https://www.wjx.top/vm/mo396H3.aspx">Call For Paper</a>
```

Comment:
I think I'd be good to use here the same type of button as we have for the rest of events, maybe we would need to update some of them that aren't following this.

```suggestion
  <div class="display-flex gap-10 margin-bottom-30">
    <a class="btn btn-primary" href="https://www.wjx.top/vm/mo396H3.aspx">Call For Papers</a>
  </div>
```